codespell_lib/data/dictionary.txt

```diff
@@ -860,7 +861,8 @@ afficionados->aficionados
 affilate->affiliate
 affilates->affiliates
 affilliate->affiliate
-affinitied->affinitized
+affinitied->affinities
+affinitized->affinity, affinities,
```

Comment:
I'd be tempted to flip these two suggestions, only because the -ies one is fewer changes typo wise than the other.

Reply:
done